main.go

```diff
@@ -552,7 +581,12 @@ func hashWith(salt, value string) string {
 	const length = 8
 
 	d := sha256.New()
-	io.WriteString(d, salt)
+
+	if len(seed) > 0 {
+		d.Write(seed)
+	} else {
+		io.WriteString(d, salt)
+	}
```

Comment:
Ok

Reply:
The buildid is not consistent between test runs.
I suspect this is because the build directory changes between testruns.
I am ok with making this change but I then cannot check the garbled identifier "teststringVar" for consistency, although in a real world scenario it would be consistent.

Reply:
Well, the test shouldn't check exactly what the final garbled identifiers/strings are. That's an implementation detail that might change at any moment.

You should test the behavior that is actually documented, such as:

* When garbling, the original name isn't present in the binary
* Doing two garble builds with different or random seeds will produce different binaries
* Doing two garble build with equal or no seeds will produce equal binaries

Reply:
Good point